client/containers/Pub/PubDocument/PubBody.js

```diff
@@ -206,11 +213,11 @@ const PubBody = (props) => {
 								firebaseRef: firebaseBranchRef,
 								clientData: props.collabData.localCollabUser,
 								initialDocKey: pubData.initialDocKey,
-								onStatusChange: debounce((status) => {
+								onStatusChange: (status) => {
 									getNextStatus(status, (nextStatus) => {
 										props.updateLocalData('collab', nextStatus);
 									});
-								}, 250),
+								},
```

Comment:
The purpose of this debounce was to prevent the status from ever flickering as it quickly toggles between saved and saving states. With it removed, this seems to happen frequently:

![flicker](https://user-images.githubusercontent.com/1000455/72429668-6a7edb80-3788-11ea-8b51-0ab9d4a564b1.gif)

If we do add a debounce back, best to use the `useDebounce` hook from `react-use` in a more appropriate place.

Reply:
Huh, this is not happening for me, but best to restore it!